src/main/java/controller/LottoController.java

```diff
@@ -0,0 +1,80 @@
+package controller;
+
+import model.*;
+import service.LottoService;
+import view.InputView;
+import view.ResultView;
+import java.util.Arrays;
+import java.util.List;
+
+public class LottoController {
+
+    private static final int TICKET_PRICE = 1000;
+    private final InputView inputView = new InputView();
+    private final ResultView resultView = new ResultView();
+    private final LottoService lottoService = new LottoService();
+
+    public void lottoRun() {
+        int purchaseAmount = getPurchaseAmount();
+        int manualCount = inputView.inputManualCount();
+        List<String> manualLottoList = getManualLottoList(manualCount);
+
+        List<Lotto> lottoList = lottoService.generateLottos(purchaseAmount, manualLottoList);
+        resultView.displayPurchasedLottoTickets(getLottoTicketStrings(lottoList), manualCount);
+
+        LottoResult lottoResult = new LottoResult(getWinningNumbers(), inputView.inputBonus());
+        List<LottoRank> lottoRanks = lottoService.calculateRank(lottoResult, lottoList);
+        printResults(lottoRanks);

```

Comment:
```suggestion
        List<Lotto> lottoList = lottoService.generateLottos(purchaseAmount, manualLottoList);
        resultView.displayPurchasedLottoTickets(formatLottoTickets(lottoList), manualCount);

        LottoResult lottoResult = createLottoResult();
        List<LottoRank> lottoRanks = lottoResult.calculateRank(lottoList);
        printResults(lottoRanks);
```

지후의 코드는 제가 봤을때 은연중에 객체간의 결합도가 높아질 가능성이 보여요
제가 제시한 코드와 지후의 코드의 차이점중 **핵심**은 **메서드 호출을 통해서 내부의 데이터에 접근하는 것이 아니라, 객체에 필요한 작업을 요청하는 방식**이라는 거에요

이에 대해서 고민해보고 지후의 생각을 알려줘요 😁😁

Reply:
코드 리뷰를 받을수록 **객체 지향**이라는 개념에 대해 더 깊이 생각하게 되는 것 같습니다. 
리뷰어님이 보여주신 것처럼 객체가 자신의 상태를 스스로 책임지고, 객체에게 작업을 요청하는 방식이
더 객체 지향적이고 자연스러워 보인다고 생각하게 되네요,

막상 코드 구현을 할 때는 각 객체가 어떤 책임을 가져야 할지 고민도 되고 많이 버벅거리는데,
그럴 때마다 객체 지향을 염두에 두고 `이 객체가 이 작업을 수행하는 것이 자연스러울까`'라고 
질문을 던지면서 구현해보도록 노력하겠습니다 !

말씀하신 대로 리팩토링을 진행하면서 점점 더 객체 지향적인 구조로 개선할 수 있었고, 좋은 인사이트를 얻을 수 있었습니다.
정말 감사드립니다! 🙌